src/main/java/io/xpring/xrpl/Utils.java

```diff
@@ -116,4 +120,97 @@ public static boolean isValidClassicAddress(String address) {
   public static String toTransactionHash(String transactionBlobHex) {
     return javaScriptUtils.toTransactionHash(transactionBlobHex);
   }
+
+  /**
+   * Convert from units in drops to xrp.
+   *
+   * @param drops An amount of xrp expressed in units of drops.
+   * @return A String representing the drops amount in units of xrp.
+   * @throws Exception if drops is in an invalid format.
+   */
+  public static String dropsToXrp(String drops) throws Exception {
+    String dropsRegex = "^-?[0-9]*['.']?[0-9]*$";
+    Pattern dropsPattern = Pattern.compile(dropsRegex);
+
+    Matcher dropsMatcher = dropsPattern.matcher(drops);
+    if (!dropsMatcher.matches()) {
+      throw new Exception(String.format(
+              "dropsToXrp: invalid value %s, should be a number matching %s.", drops, dropsRegex));
+    } else if (drops == ".") {
+      throw new Exception(String.format(
+              "dropsToXrp: invalid value %s, should be a BigNumber or string-encoded number.", drops));
+    }
+
+    // Converting to BigInteger and then back to string should remove any
+    // decimal point followed by zeros, e.g. '1.00', which is the only valid decimal
+    // representation of drops, which must be whole numbers.
+    // Important: specify base 10 to avoid exponential notation, e.g. '1e-7'
+    drops = new BigDecimal(drops).toBigIntegerExact().toString(10);
+
+    // drops are only whole units
+    if (drops.contains(".")) {
+      throw new Exception(String.format("dropsToXrp: value %s has too many decimal places.", drops));
+    }
+
+    // This should never happen; the value has already been
+    // validated above. This just ensures BigNumber did not do
+    // something unexpected.
+    if (!dropsMatcher.matches()) {
+      throw new Exception(String.format(
+              "dropsToXrp: failed sanity check - value %s does not match %s.", drops, dropsRegex));
+    }
+
+    return new BigDecimal(drops).divide(new BigDecimal(1000000.0)).toPlainString();
+  }
+
+  /**
+   * Convert from units in xrp to drops.
+   *
+   * @param xrp An amount of xrp expressed in units of xrp.
+   * @return A String representing an amount of xrp expressed in units of drops.
+   * @throws Exception if xrp is in invalid format.
+   */
+  public static String xrpToDrops(String xrp) throws Exception {
```

Comment:
Likewise I would change this method to throw `IllegalArgumentException` instead of the more generic `Exception` type.